bin/install.sh

```diff
@@ -166,7 +166,11 @@ julia --color=yes -e "using InteractiveUtils; versioninfo()"
 ### Install utilities
 
 # Throw out trailing .jl, assume the name is otherwise a valid Julia package name
-JLPKG="$(echo "${CIRRUS_REPO_NAME}" | cut -d'.' -f 1)"
+if [ -z "${JULIA_PROJECT_SUBDIR}" ]; then
+    JLPKG="$(echo "${CIRRUS_REPO_NAME}" | cut -d'.' -f 1)"
+else
+    JLPKG="$(echo "${JULIA_PROJECT_SUBDIR}" | cut -d'.' -f 1)"
```

Comment:
If the path to the project contains a `.` or is more than one level deep, this won't work as expected. An example of the latter that comes to mind is ArrowTypes, which lives in `arrow-julia/src/ArrowTypes`. I think what we want here is something like
```sh
JLPKG="$(basename "${CIRRUS_REPO_NAME}/${JULIA_PROJECT_SUBDIR}" | cut -d'.' -f 1)"
```
That should work regardless of whether `JULIA_PROJECT_SUBDIR` is present.